src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="info" action="ttps://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    name="user_info">
```

Comment:
You should be consistent in your formatting. If you move attributes to a new line - each of them should take a separate line. 
Also, don't forget about adding **exactly 2 indents** for each level of nesting

```suggestion
    <form 
      class="info" 
      action="ttps://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="user_info"
    >
```